setup.py

```diff
@@ -126,6 +126,7 @@
             'flocker-reportstate = flocker.node.script:flocker_reportstate_main',
             'flocker-zfs-agent = flocker.node.script:flocker_zfs_agent_main',
             'flocker-control = flocker.control.script:flocker_control_main',
+            'flocker = flocker.cli.script:flocker_cli_main',
```

Comment:
I'm guessing that will be a follow up issue, please put an "XXX" and a link to that here if so.